core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
I think it is a consequence of the changes to the request from this KIP though, right? Before we would always have at least one partition in the request, so we always had somewhere to pack an error code in the response. Now that's no longer true.